WooCommerce/Classes/POS/Presentation/CardReaderConnection/UI States/Reader Messages/PointOfSaleCardPresentPaymentValidatingOrderMessageView.swift

```diff
@@ -1,13 +1,41 @@
 import SwiftUI
 
 struct PointOfSaleCardPresentPaymentValidatingOrderMessageView: View {
-    let viewModel: PointOfSaleCardPresentPaymentValidatingOrderMessageViewModel
+    let title: String
+    let message: String
+
+    init(title: String, message: String) {
+        self.title = title
+        self.message = message
+    }
+
+    init(viewModel: PointOfSaleCardPresentPaymentValidatingOrderMessageViewModel) {
+        self.title = viewModel.title
+        self.message = viewModel.message
+    }
```

Comment:
Having one or the other of these seems like it would be enough... I don't have a strong opinion which we should keep though. If it's the viewmodel, let's rename it to match the `ActivityIndicating` suggestion too.